style/styles_footer.css

```diff
@@ -0,0 +1,363 @@
+.footer {
+  background-color: #000;
+  color: #fff;
+  width: 100%;
+}
+
+.footer__block {
+  display: flex;
+  padding-top: 70px;
+  padding-left: 300px;
+}
+
+.footer-navigation__link {
+  text-decoration: none;
+  color: #fff;
+  font-family: "Gotham Pro Regular", sans-serif;
+  font-size: 12px;
+  padding-bottom: 6px;
+}
+
+.footer-navigation__title {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  margin: 0;
+  padding-bottom: 10px;
+}
+
+.footer-navigation {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.footer-sign__button {
+  background-color: #000;
+  color: #fff;
+  width: 136.57px;
+  height: 29.7px;
+  font-family: "Gotham Pro Bold";
+  font-size: 10px;
+  border: 1.5px solid #fff;
+  padding-top: 4px;
+}
+
+.footer-sign {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-sign__title {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  margin: 0;
+  padding-bottom: 5px;
+}
+
+.footer-sign__text {
+  color: #fff;
+  font-family: "Gotham Pro Regular", sans-serif;
+  font-size: 12px;
+  padding-bottom: 12px;
+}
+
+.footer__social {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 137px;
+  padding-top: 34px;
+}
+
+.copyright {
+  font-family: "Gotham Pro Regular", sans-serif;
+  font-size: 12px;
+  padding-left: 484px;
+  padding-top: 51px;
+  padding-bottom: 33px;
+  font-style: normal;
+}
+
+.navigation_one {
+  margin-right: 114px;
+}
+
+.navigation_two {
+  margin-right: 72px;
+}
+
+.navigation_three {
+  margin-right: 86px;
+}
+
+@media (max-width: 1170px) {
+  .footer__block {
+    padding-left: 200px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .footer__block {
+    padding-left: 127px;
+  }
+
+  .footer-sign__title {
+    padding-bottom: 6px;
+  }
+
+  .footer-sign__button {
+    padding-top: 5px;
+  }
+
+  .copyright {
+    padding-left: 410px;
+    padding-top: 52px;
+  }
+}
+
+@media (max-width: 768px) {
+  .footer__block {
+    padding-left: 51px;
+    padding-top: 60px;
+  }
+
+  .footer-navigation__title {
+    white-space: nowrap;
+    padding-bottom: 7px;
+  }
+
+  .navigation_one {
+    margin-right: 56px;
+  }
+
+  .navigation_two {
+    margin-right: 65px;
+  }
+
+  .navigation_three {
+    margin-right: 70px;
+  }
+
+  .footer-sign {
+    width: 185px;
+  }
+
+  .footer-sign__title {
+    padding-bottom: 3px;
+  }
+
+  .footer-sign__text {
+    word-spacing: 5px;
+    padding-bottom: 11px;
+  }
+
+  .footer-sign__button {
+    width: 143.23px;
+    height: 25.68px;
+  }
+
+  .footer__social {
+    padding-top: 28px;
+    width: 116px;
+    align-items: flex-end;
+  }
+
+  .facebook {
+    width: 6.92px;
+    height: 13.83;
+  }
+
+  .twitter {
+    width: 15.56px;
+    height: 12.64px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)